package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "devmountian_capstone",
+	"version": "1.0.0",
+	"description": "",
+	"main": "calculator.js",
+	"scripts": {
+		"start": "node ./server.js"
```

Comment:
Changed our start script to run `node ./server.js` instead of the [http-server](https://www.npmjs.com/package/http-server) package like we had before.